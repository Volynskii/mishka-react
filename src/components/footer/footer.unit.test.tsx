import React from "react";
import Footer from "./footer";
import renderer from "react-test-renderer";


describe('Footer component', ()=> {
    it('should render Footer Component', ()=> {
        const dom = <Footer />;
        const component = renderer.create(dom);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
