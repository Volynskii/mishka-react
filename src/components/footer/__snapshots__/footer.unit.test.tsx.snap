// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer Component 1`] = `
<article
  className="footer"
>
  <section
    className="footer__wrapper"
  >
    <a
      className="footer__wrapper__logotype"
      href="#navigation"
    >
       
    </a>
    <div
      className="footer__wrapper__social-media"
    >
      <a
        className="footer__wrapper__social-media__instagram"
        href="https://www.instagram.com/?hl=ru"
      >
         
      </a>
      <a
        className="footer__wrapper__social-media__facebook"
        href="https://www.facebook.com/"
      >
         
      </a>
      <a
        className="footer__wrapper__social-media__twitter"
        href="https://www.twitter.com/"
      >
         
      </a>
    </div>
    <ul
      className="copyright"
    >
      <li
        className="copyright__name"
      >
        Разработано
      </li>
      <li
        className="copyright__logotype"
      >
         
      </li>
    </ul>
  </section>
</article>
`;
